versioning_function.sql

```diff
@@ -68,21 +67,6 @@ BEGIN
       RETURN NEW;
     END IF;
 
-    SELECT current_setting('server_version_num')::integer
-    INTO pg_version;
-
-    -- to support postgres < 9.6
-    IF pg_version < 90600 THEN
-      -- check if history table exits
-      IF to_regclass(history_table::cstring) IS NULL THEN
-        RAISE 'relation "%" does not exist', history_table;
-      END IF;
-    ELSE
-      IF to_regclass(history_table) IS NULL THEN
-        RAISE 'relation "%" does not exist', history_table;
-      END IF;
```

Comment:
I guess this part should be kept around?

Reply:
I didn't see much harm at first sight, but we can keep it for safety.

Reply:
I also have limited access to this repo and can't close this pr. @msand can you close it?